### PR TITLE
[1LP][RFR] Fixed vsphere skiping issue

### PR DESCRIPTION
**Pull request by ganeshhubale**
- This PR fixes the issue of skipped test cases related to vsphere provider

### CODE REVIEW

**Comment by ganeshhubale** [comment]
One test case failed on 5.11 which is not related the changes made in this PR. Hence moved to RFR.

File: cfme/tests/automate/test_git_import.py [hunk 1]
```diff
@@ -45,7 +45,8 @@ def new_user(appliance):
         value_assign="Database",
     )
     yield user
-    user.delete_if_exists()
+    user = appliance.rest_api.collections.users.get(name=user.name)
+    user.action.delete()
```
Comment (digitronik) [inline cfme/tests/automate/test_git_import.py]: @ganeshhubale check exists with rest and then delete... but the reference of BZ as well

Comment (ganeshhubale) [reply cfme/tests/automate/test_git_import.py]: Actually I am keeping these changes as it was (```user.delete_if_exists()```) [this issue](https://bugzilla.redhat.com/show_bug.cgi?id=1720273) got fixed in 5.11.0.11

**Comment by ganeshhubale** [comment]
> One test case failed on 5.11 which is not related the changes made in this PR. Hence moved to RFR.

This test case is passing locally.

**Comment by mshriver** [comment]
PRT behavior does not appear related to the collection change here.

**Pull request merged**